docs/compute-engine/model_optimization_guide.md

```diff
@@ -0,0 +1,108 @@
+# Optimizing models for Larq Compute Engine
+
+Larq Compute Engine is capable of running any Larq model, but some model
+architectures will perform better than others. These are some important
+guidelines to follow when designing models to be run on LCE, so as to ensure
+that inference is as fast as possible.
+
+!!! important
+    This guide assumes that you are deploying Larq Compute Engine on an Arm64
+    device, such as an Android phone or a Raspberry Pi 4 board (with 64-bit OS).
+
+*   **Use multiples of 32 input channels**
+
+    The LCE binary ops (binary convolution and binary maxpool) operate on
+    multiples of 32 input channels. Before a binary op, the input channels are
+    padded out to a multiple of 32, which means that a binary convolution
+    applied to an input with 16 channels and a binary convolution applied to an
+    input with 32 channels will run equally fast.
+
+*   **Use multiples of 8 output channels**
+
+    The LCE optimised binary convolution op computes the result of 8 output
+    channels in parallel, so if the number of output channels is not a multiple
+    of 8 there will be wasted computation.
+
+*   **Use `QuantConv2D` → `ReLU` → `BatchNormalization` to take advantage of op
+    fusing**
```

Comment:
How do you feel about changing the title to something like this:
```suggestion
*   **Use `QuantConv2D` → `ReLU` → `BatchNormalization` instead of `QuantConv2D` → `BatchNormalization` → `ReLU` to take advantage of op fusing**
```

While the text explains it great, to me the title suggests that I should always use a relu between bconv and batch norm but it would still be fine to not use a relu at all.